src/main/resources/latest-spark-swagger-output.yaml

```diff
@@ -859,6 +878,21 @@ definitions:
       lat:
         type: "number"
         format: "double"
+  ItineraryExistenceResult:
+    type: "object"
+    properties:
+      invalidDates:
+        type: "array"
+        items:
+          type: "string"
+      validDates:
+        type: "array"
+        items:
+          type: "string"
+      itineraries:
```

Comment:
Not a big deal for me, but the `isValid` JSON prop is not showing up here.

Reply:
Same reason as for the `itineraries` field. Although there is a getter and setter,`valid` is not a field, so spark-swagger just ignores it.

Reply:
I added the field in the public docs in https://github.com/ibi-group/otp-middleware/pull/77/commits/d26baf619770dce9e11667961912b80261de090c.